File: Coding/le8.py
from array import array

arr = array("i", [1, 2, 3, 4])


def rev():
    m = 1
    i = m
    j = len(arr) - 1
    while i < j:
        temp = arr[i]
        arr[i] = arr[j]
        arr[j] = temp
        i = i + 1
        j = j - 1
    for i in range(0, len(arr)):
        print(arr[i])


rev()

File: Coding/le8.cpp
#include <bits/stdc++.h>
using namespace std;
void rev()
{
    int arr[] = {1, 2, 3, 4, 44};
    int m = 0;
    int j = 4;
    for (int i = m; i < j; i++)
    {
        swap(arr[i], arr[j--]);
    }
    for (int i : arr)
    {
        cout << i << endl;
    }
}
int main()
{
    rev();
    return 0;
}